### DISCUSSION TITLE
Cable and adapters that work to extend the uctronics PoE adapter

### DISCUSSION THREAD

**Discussion by donsandrop** [gtxaspec/wz_mini_hacks#521] [Q&A]
Wanted to see if anyone has had better luck than me getting an extension on the uctronics adapter.

I use the [uctronics adapter](https://www.amazon.com/dp/B0B9ZLVGJ4?psc=1&ref=ppx_yo2ov_dt_b_product_details) for my PoE then I use a [USB micro to USB A adapter](https://www.amazon.com/dp/B07GJJLNX7?psc=1&ref=ppx_yo2ov_dt_b_product_details) and then I connect to a 3' or 10' (approx) monoprice cable. Unfortunately this never connects unless I switch to the 6" Wyze cable.

Does anyone have a combination with a 3' or longer cable that works. Can you share the links you used?
Thank you in advance

**Comment by Bob-AOL** [comment]
This has been discussed before. Maybe over in the LAN adapter thread? Can't recall  solution.

Of course the question becomes why not just run UTP all the way to the camera and be done with it instead of messing around with adapters and extra spaghetti.

**Reply by donsandrop** [reply]
Hi Bob,
It's been discussed but looking for specific brands of cables and if anyone has had luck with one brand over another.
I have UTP running 50+ feet to the uctronics PoE adapter and then I would like about 3 feet of USB cable so I can make it to the outside from the attic without crawling over attic trusses. 

Thanks!

**Reply by Bob-AOL** [reply]
Got it.

So the issue is actually the USB connectorization and finding the correct implementation of the USB standard for the application, not the cable -- provided it has four (perhaps five) conductors. The link below is a comprehensive resource. You should be able to determine the magic combination for your needs with a little reading..., and maybe some re-reading. LOL

https://www.newnex.com/usb-connector-type-guide.php
